src/main/java/no/unit/nva/model/instancetypes/journal/JournalArticle.java

```diff
@@ -12,7 +13,9 @@
     private String volume;
     private String issue;
     private String articleNumber;
-    private JournalArticleContentType content;
+
+    @JsonAlias("content") //for backwards compatibility.
```

Comment:
Renamed for compatibility with the other Publication instances. Should have been in another PR, but slipped into this one.